src/abap/pretty_printer.ts

```diff
@@ -32,15 +32,24 @@ class Stack {
 }
 
 class Indentation {
-// returns list of expected indentation for each line/statement?
-  public static run(file: ABAPFile, options: IndentationOptions): number[] {
+  private file: ABAPFile;
+  private options: IndentationOptions;
+  private globalClasses = new Set();
+
+  constructor(file: ABAPFile, options: IndentationOptions) {
+    this.file = file;
+    this.options = options;
+  }
+
+  // returns list of expected indentation for each line/statement?
+  public run(): number[] {
```

Comment:
suggest moving `file` parameter from `constructor` to `run`

Reply:
done